src/binding/petsc4py/docs/source/petsc_python_types.rst

```diff
@@ -1,8 +1,33 @@
+.. _petsc_python_types:
+
+PETSc Python types
+==================
+
+Here we discuss details about Python-aware PETSc types that can be used within the library.
+
 .. _petsc_python_mat:
 
 PETSc Python matrix type
 ========================
 
+PETSc provides a convenient way to compute the action of linear operators coded
+in Python through the MATPYTHON class.
```

Comment:
Not sure if this patch will actually work
```suggestion
in Python through the `~petsc4py.PETSc.Mat.Type.PYTHON` `Mat` subtype.
```

Reply:
/Users/szampini/Devel/petsc_petsc4py_sprint/src/binding/petsc4py/docs/source/petsc_python_types.rst:13: WARNING: 'any' reference target not found: ~petsc4py.PETSc.Mat.Type.PYTHON